tools/tertiary-analysis/cell-types-analysis/ct_get_tool_perf_table.xml

```diff
@@ -42,14 +45,15 @@
         <param type="data" name="input_files" label="Input tables" multiple="true" format="txt" help="Classification tables obtained from multiple classisifiers" />
         <param type="data" name="ref_file" label="Reference metadata file" format="txt" help="Metadata file with reference labels" />
         <param type="boolean" name="parallel" label="Parallel execution" checked="false" help="Should the table values be computed in parallel?" />
+        <param type="boolean" name="include_sem_siml" label="Include Semantic Similarity" checked="false" help="Should semantic similarity be included in calculation of combined score?" />
         <param type="data" name="exclusions" label="Exclusions" format="yml" optional="true" help="Config file with exclusions" />
         <param type="data" name="ontology_graph" label="Ontology graph" format="obo" help="Ontology graph to run semantis similarity computations" />
         <param type="data" name="lab_cl_mapping" label="Label - CL term mapping" format="rdata" help="Mapping between cell labels and ontology terms" />
         <param type="text" name="barcode_col_ref" label="Reference file barcode field" value='cell_id' help="Name of barcode field in reference metadata file" />
         <param type="text" name="barcode_col_pred" label="Predicted file barcode field" value="cell_id" help="Name of barcode field in predictions file" />
         <param type="text" name="label_col_ref" label="Reference label column" value="cell_type" help="Label column in reference file" />
         <param type="text" name="label_col_pred" label="Predicted file label column" value="pred_label" help="Name of label field in predictions file" />
-        <param type="text" name="semantic_sim_metric" label="Semantic similarity metric" value="edge_resnik" help="Type of semantic similarity metric used" />
+        <param type="text" name="semantic_sim_metric" label="Semantic similarity metric" value="lin" help="Type of semantic similarity metric used" />
```

Comment:
If you can pick among values here, which I presume you can, this should be select element showing all possible values (if they are a reasonable amount). Otherwise the user will need to know their names as used inside the code, which is difficult.

Reply:
@pcm32 there are 12 different metrics, it's probably easier to refer the user to the list of them.

Reply:
Yes, but that still exposes us to users asking why it didn't work when they mis typed the name of the metric. Believe me, it will save precious time of your life not having to answer those questions again and again. An alternative is to make the values self completable. But 12 for a select is not a lot, so I would go for that.

Reply:
@okay, will add it. Can I add these options as macros so as not to copy things over multiple times?

Reply:
yes, I think macros would be the way to go here.